src/main/java/com/epam/reportportal/junit/ReportPortalListener.java

```diff
@@ -320,7 +324,14 @@ protected void stopRunner(@Nonnull final Object runner) {
 			l.setStatus(status);
 			if (l.getParentId() == null) {
 				rq.setStatus(ofNullable(status).map(Enum::name).orElse(null));
+				if (!failedTestsMessages.isEmpty()) {
```

Comment:
You don't need to add errors on Suites

Reply:
Updated, I removed it from Suites level, it was discussed with Andrey.